src/main/java/it/gov/pagopa/reward/notification/dto/mapper/detail/RewardsNotification2ExportDetailDTOMapper.java

```diff
@@ -0,0 +1,19 @@
+package it.gov.pagopa.reward.notification.dto.mapper.detail;
+
+import it.gov.pagopa.reward.notification.dto.controller.detail.RewardNotificationDTO;
+import it.gov.pagopa.reward.notification.model.RewardsNotification;
+import it.gov.pagopa.reward.notification.utils.Utils;
+import org.springframework.stereotype.Service;
+
+@Service
+public class RewardsNotification2ExportDetailDTOMapper {
+
+    public RewardNotificationDTO apply(RewardsNotification notification) {
+        return RewardNotificationDTO.builder()
+                .id(notification.getExternalId())
+                .iban(notification.getIban())
+                .amount(Utils.cents2Eur(notification.getRewardCents()))
```

Comment:
there was a need I see to format BigDecimal:

it.gov.pagopa.reward.notification.dto.mapper.RewardOrganizationExports2ExportsDTOMapper#centsToEur

let's move that method into Utils (changing name into something like formatCents2Euro